functions/exercise20/script.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function getStringIdOfRandomChars(specifiedLength) {
+    const stringCharacters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+    let getStringIdOfRandomChars = "";
+    for (let i = 0; i < specifiedLength; i++) {
+        getStringIdOfRandomChars +=
+            stringCharacters[(Math.floor(Math.random()
+                * stringCharacters.length))];
```

Comment:
Please notice, that you do this operation: stringCharacters.length (getting the length of the array) many times in a loop. For better peformance you should get the length before the loop and refer to the result in the loop to avoid calling length multiple times.